Chatify/chat/templates/chat/login.html

```diff
@@ -5,21 +5,27 @@
   <head>
     <meta charset="utf-8">
    <!----<title>Login Form Design | CodeLab</title>---->
-    <link rel="stylesheet" href="{% static 'css/register.css' %}">
```

Comment:
create a base.HTML and put all basic designs and also all JS includes should be in that HTML

then you can extend base.html in your child HTML and use content block from base HTML